code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,6 +319,22 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
+	if(method=INGEST && is_species(M, /datum/species/squid))
+		to_chat(M, "<span class='danger'>Your tongue shrivels as you taste the salt! It burns!</span>")
+		M.adjustFireLoss(5, TRUE)
+		sleep(20)
```

Comment:
This isn't a loop, this sleep does nothing.

Reply:
Interesting. Thought this was to apply sleep(time)

Reply:
Yes but in this instance sleeping does nothing.

Reply:
> are you even testing this code as you write it? because it shouldn't even compile
Ye, sorry, knee deep in just trying to implement things. Getting an inconsistent indentation on squidpeople.dm